templates/eap74-beta-sso-s2i.json

```diff
@@ -46,7 +46,7 @@
             "displayName": "SSO Image Name",
             "description": "Name of the SSO image to use, examples: redhat-sso74-openshift:latest",
             "name": "SSO_IMAGE_NAME",
-            "value": "redhat-sso74-openshift:latest",
+            "value": "sso74-openshift-rhel8:7.4.5.GA",
```

Comment:
Do we want to fix version to specific version?
What am I missing? I see just `latest` [1] and `7.4` [2] tags there.

[1] https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso74-dev/templates/sso74-image-stream.json#L30
[2] https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso74-dev/templates/sso74-image-stream.json#L37

Reply:
@mchoma, thanks for spotting that -- I'd looked at their imagestreams right before doing this and I didn't see the latest tag :S
In any case, we should just use that, and assume the user has imported the latest imagestreams from whatever sso tag they want to use (we shouldn't rely on that sso -dev branch being available in the registry yet, but the tags should be.)